### ISSUE TITLE
xl2tpd does not reconnect

### ISSUE THREAD

**Issue by uk3** [openwrt/packages#1661]
Hello,

I observed the following problem in current trunk: When my l2tp tunnel gets disconnected for some reason, it does not reconnect automatically.

Only after a click in the gui the tunnel reconnects (immediately without problems).

**Comment by yousong** [comment]
Duplicate of #1334 .  Close it for now.
